content/intro-to-storybook/svelte/es/test.md

```diff
@@ -0,0 +1,144 @@
+
+## Pruebas de regresión visual para Storybook
+
+Las pruebas de regresión visual están diseñadas para detectar cambios en la apariencia. Funcionan haciendo capturas de pantalla de todas las historias y comparando con los cambios superficiales. Esto es perfecto para verificar elementos gráficos como diseño, color, tamaño y contraste.
+
+<video autoPlay muted playsInline loop style="width:480px; margin: 0 auto;">
+  <source
+    src="/intro-to-storybook/visual-regression-testing.mp4"
+    type="video/mp4"
+  />
+</video>
+
+Storybook es una herramienta fantástica para las pruebas de regresión visual porque cada historia es esencialmente una especificación del test. Cada vez que escribimos o actualizamos una historia recibimos una especificación gratis!
+
+Existen varias herramientas para la prueba de regresión visual. Para equipos profesionales recomendamos [**Chromatic**](https://www.chromatic.com/), un complemento hecho por las personas de Storybook que ejecuta pruebas en la nube.
+
+## Configurar pruebas de regresión visual
+
+Chromatic es un complemento de Storybook para pruebas de regresión visual y revisión en la nube. Dado que es un servicio de pago (con una prueba gratuita), puede que no sea para todos. Sin embargo, Chromatic es un ejemplo instructivo de un flujo de trabajo de pruebas visuales de producción que probaremos gratuitamente. Echemos un vistazo.
+
+### Inicializar repositorio local
+
+En esta etapa todavía no hay una configuración de repositorio todavía. Cambiemos eso y configurémoslo:
+
+```bash
+$ git init
+```
+
+A continuación, agregue archivos al primer commit.
+
+```bash
+$ git add -A
+```
+
+Ahora haz commit de los archivos.
+
+```bash
+$ git commit -m "taskbox UI"
+```
+
+### Añadiendo Chromatic
+
+Agregando el paquete como una dependencia.
+
+```bash
+npm install -D chromatic
+```
+
+Ahora [logueate en Chromatic](https://www.chromatic.com/start) con tú cuenta de GitHub (Chromatic solo te pedirá algunos permisos simples). Crea un proyecto con nombre "taskbox" y copia tu `project-token` único.
+
+<video autoPlay muted playsInline loop style="width:520px; margin: 0 auto;">
+  <source
+    src="/intro-to-storybook/chromatic-setup-learnstorybook.mp4"
+    type="video/mp4"
+  />
+</video>
+
+Se requiere un pequeño cambio en el script `build-storybook` para permitir que el complemento `chromatic` muestre correctamente tanto los recursos (iconos y fuentes) como el css que se agregó al comienzo del tutorial.
+
+```json
+{
+  ....
+  "scripts":{
+    ...
+    "build-storybook": "build-storybook -s public",
+  }
+}
+```
+
+Ejecuta el comando de prueba en la línea de comandos para configurar las pruebas de regresión visual para Storybook. No olvides añadir tu código de aplicación único en el `<project-token>`.
+
+```bash
+ npx chromatic --project-token=<project-token>
+```
+
+</div>
+
```

Comment:
@IvanSotelo if you can take a look at #319 for the text that needs to be added before "Una vez el primer test esté completo, " @ghengeveld introduced some changes in the text that are already in play, but they needed to be fined tuned. If you're ok with the wording, incorporate them, otherwise i would like for you to propose a more suitable wording to go here and i'll adjust accordingly in the other pr. Sounds good?